### PR TITLE
Grau 50

### PR DESCRIPTION
**Pull request by DanMaguireData**

### CODE REVIEW

File: grau/blueprints/bodypart/routes.py [hunk 1]
```diff
@@ -40,3 +40,34 @@ def get_bodypart(bodypart_id: int):
...
+                         $ref: '#/components/schemas/BodypartQuerySchema'
+        security:
+            - cookieAuth: []
+        responses:
+            '204':
+                description: Bodyparts found successfully
+            '400':
+                description: Query validation error
+    """
+    return jsonify(
```
Comment (DANLENEHAN) [inline grau/blueprints/bodypart/routes.py:69]: I think we should add the jsonifying to 'api_response_handler' versus doing it here. Currently it looks like the below:

`        return (
            jsonify(self.response_body)
            if isinstance(self.response_body, dict)
            else self.response_body,
            status_code.value,
        )`

We should really jsonify for any non-primitive data structures.

**Review by DANLENEHAN — COMMENTED** [review]
Can you have a GET and POST for the same "/get" defined as two completely different route functions. I'm guessing since all the tests passed you can? I know you can have to request types in the same route function e.g. methods=["GET", "POST"] but didn't think of splitting them into two separate ones. I guess maybe they're different because the other gets are "get/id"

File: grau/blueprints/bodypart/routes.py (hunk 1 above)
Comment (DanMaguireData) [reply grau/blueprints/bodypart/routes.py]: Fixed this in latest PR

**Comment by DanMaguireData** [comment]
It must be able to differentiate that they are different paths based on the /{id}, it all seems to work in the tests and swagger so hoping its g, if theres a pattern you'd prefer happy to go with whatever

**Review by DANLENEHAN — APPROVED** [review]

**Pull request merged**